manifests/04_clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: aws-ebs-csi-driver-operator-clusterrole
+subjects:
+  - kind: ServiceAccount
+    name: aws-ebs-csi-driver-operator
+    namespace: openshift-cluster-csi-drivers
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
This is unfortunate. But we can fix it as a subsequent PR, what do you think?

Reply:
Fixed